src/Summoner/CLI.hs

```diff
@@ -41,6 +46,31 @@ summon = execParser prsr >>= runCommand
 runCommand :: Command -> IO ()
 runCommand = \case
     New opts -> runNew opts
+    ShowInfo opts -> runShow opts
+
+runShow :: ShowOpts -> IO ()
+runShow opts =
+    case opts of
+        -- show list of all available GHC versions
+        GhcList -> ghcVers
+        -- show a list of all available licenses
+        LicenseList -> licenses
+            -- show a specific license
+        LicenseText name ->
+            case parseLicenseName (toText name) of
+                Nothing -> do
+                    errorMessage "This wasn't a valid choice."
+                    infoMessage "Here is the list of supported licenses"
+                    licenses
+                Just licenseName -> do
+                    -- get and show a license`s text
+                    mLicenseText <- getLicense licenseName
+                    case mLicenseText :: Maybe License of
+                        Just t  -> putStr $ unLicense t
+                        Nothing -> error "Broken predefined license list"
+  where
+    ghcVers = mapM_ (successMessage . T.append "\10148 " . showGhcVer) (reverse universe)
+    licenses = mapM_ (successMessage . (\ x -> T.append "\10148 " $ show (x :: LicenseName))) universe
```

Comment:
It's always better to write explicit type of functions inside `where` instead of specifying types of variables locally like `x :: LicenseName`.

Reply:
`licenses`  has type `IO ()`. It doesn`t explicit what is inside. Do you mean something another?

Reply:
Even if it's `IO` still better to write everything explicitly. We even have a note on it in the style guide.

Reply:
> Even if it's IO still better to write everything explicitly.

:+1: 

> We even have a note on it in the style guide.

Could you show me the style guide? I couldn`t find the note here https://github.com/kowainik/org

Reply:
Yes, sure. Currently, we are following [this style guide](https://github.com/serokell/serokell-util/blob/master/serokell-style.md#serokell-haskell-style-guide).

Sorry, the link in our `org` is broken :disappointed:, I will fix it!